unstructured_ingest/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.26-dev4"  # pragma: no cover
+__version__ = "0.0.30"  # pragma: no cover
```

Comment:
how did it go from 26-dev4 to 30? Also please use dev versions unless it is a release.